tools/build-release.py

```diff
@@ -41,19 +42,16 @@ def build_release():
     release_base = os.path.join(staging_dir, release_package_name)
     release_top_path = os.path.join(os.path.dirname(tools_dir),
         "release-top-level-artifacts")
+    helm_chart_path = os.path.join(os.path.dirname(tools_dir),
+        "helm-charts")
```

Comment:
the changes for this file seems not enough.
because the helm charts file in this dir is hard-coded to 0.8.
can we add a folder for all helm-chart files, and each released version can have one on its own?

Reply:
I don't think we should include the helm charts for all versions, since the released ones can be installed without having this files. 
I think the better solution is to change 0.8.0 version from the helm charts to latest. When testing it, we will need to change the image version to the locally built ones as mentioned in the README: https://github.com/apache/incubator-yunikorn-release/blob/master/release-top-level-artifacts/README.md

Reply:
OK, I am good with setting it to the latest.
But when we release a version, e.g next 0.9.0. Our script will copy the helm charts to the release-dir, then our script should be able to replace `latest` to the actual version we are going to release, i.e 0.9.0.

Reply:
I changed the version to latest to the helm charts and implemented a version replace for the  charts that will be packaged. It will pickup the version from the config file and replace the values from values.yaml and Chart.yaml. Also I removed the index.yaml file, since we don't need it to master branch.